docker-cloud/builds/automated-build.md

```diff
@@ -105,6 +105,21 @@ run an initial test.
     Docker Cloud on every push. Only pushes to branches that are listed as the
     source for one or more tags will trigger a build.
 
+### Environment variables for builds
+
+You can set environment variables used in your build either by declaring them in
```

Comment:
@caervs I don't think it is right in any case, it is an unsecure workaround. I think we shouldn't document that practice at all (which is also unrelated to Docker Cloud). If we talk about it, it should be to show the limitations of this practice and explain the solution using build env vars.